megazords/ios/MozillaAppServicesTests/FxAccountManagerTests.swift

```diff
@@ -243,7 +243,7 @@ class FxAccountManagerTests: XCTestCase {
         let beginAuthDone = expectation(description: "beginAuthDone")
         var authURL: String?
         mgr.initialize { _ in
-            mgr.beginAuthentication { url in
+            mgr.beginAuthentication(entrypoint: "testNewAccountLogIn") { url in
```

Comment:
can these be snake case? I don't think we want to show a bad example where entrypoint format is camelCase